app/models/candidate.rb

```diff
@@ -48,7 +48,11 @@ def current_application
   end
 
   def current_application_choice
-    current_application.application_choices.order(:created_at).last
+    current_application_choices.last
+  end
+
+  def current_application_choices
+    current_application.application_choices.order(:created_at)
   end
```

Comment:
Is the order needed here? Could be left off perhaps, and there anywhere that needs a certain order could add it?

Then perhaps could use `delegate`?

```ruby
  delegate :application_choices, to: :current_application, prefix: true
```

(Just a suggestion though, feel free to ignore! 😄)

Reply:
> Is the order needed here? Could be left off perhaps, and there anywhere that needs a certain order could add it?
order isn't necessary anymore, I've removed it 👍 

I don't think we need the delegation as we already have access to application choices on current application